.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
you can prefix positions args with an underscore like `_token` but things are much trickier for kwargs

```ruby
def do_it(cow, dog: nil)
  puts "cow: #{cow}, dog: #{dog}"
end
do_it('moo', dog: 'woof')
# cow: moo, dog: woof

def just_do_it(cow)
  puts "cow: #{cow}, dog: #{dog}"
end
just_do_it('moo', dog: 'woof')
ArgumentError: wrong number of arguments (given 2, expected 1)
```
you can define it as `def just_do_it(cow, _options)` but that's pretty unclear both in terms of figuring out what might be passed by reading the signature and figuring out what arguments might be expected from reading a call site